### DISCUSSION TITLE
Is there any way to check the fan speed set in the bios?

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#448] [Q&A] [answered]
Hello. I'm using a G15 2022 model with a lot of issues.

G-Helper works perfectly, but I've always felt like the fan noise is louder when using G-Helper.

After reinstalling the armory crate and comparing in relatively the same environment and temperature, the fan actually spins about 1,000 rpm faster when using the G-Helper (this is not an exact measurement).

I'm desperate to get out of this armory crate, but is there any way to find out what the fan speed setting is in the bios?

Thanks.

**Comment by seerge** [comment]
@cheonmux generally speaking all 3 profiles are baked in bios (you can set them w/o any app just with a powershell command even)
And they come with some fan curve (for G15 i don't know how it looks like + it seem that bios is not reporting it in your case)

Do you use "default" profiles? or apply fan curve (i.e. is "apply" checked in fans + power) ?

**Reply by cheonmux** [reply]
Hi, @seerge  Thanks for answering.

I only use the "default" profile. I haven't applied any fan curves.

**Reply by seerge** [reply, marked as answer]
@cheonmux then they should be same :) unless armory secretly applies custom fan curve with each mode. If so - try to mimic / build custom curve in g-helper and also apply it ? :)

**Reply by cheonmux** [reply]
Hi there,

I downgraded my G15 bios from 313 to 311, and it seems to have solved the problem!

Thanks a lot

**Reply by seerge** [reply]
@cheonmux glad to hear :) Is it something you would recommend to other G15 users having issues with fans ? (i have seen bunch of "issues" here posted mostly G15 users having similar problems to yours)

**Reply by cheonmux** [reply]
Hi, @seerge.

Well, it only takes 5 minutes to roll back the bios, so I think it's worth a try.

Let me explain a little more about my situation, as it might help other users.

I'm using a G15 503RM 2022 (AMD 6800HS + Geforce 3060) model, and here are the issues I'm having with the 313 bios and G-Helper

1) Reboot on wake up after hibernation
2) GPU fan does not spin
3) Fan speed is high for the temperature

And the reason I decided to downgrade the bios was because I came across this post on reddit "[https://www.reddit.com/r/ZephyrusG15/comments/13l46i7/best_bios_for_2022_model/](https://www.reddit.com/r/ZephyrusG15/comments/13l46i7/best_bios_for_2022_model/)".

One of the users in the post had the same symptoms as me, and when I searched for the 313 bios on reddit, I saw that many users were experiencing various issues.

After downgrading to the 311 bios, I'm fairly confident that there is something wrong with the 313 bios.

I realize that not all users will be like me, but if it were me, I'd give it a try, with Windows automatic updates turned off.

Finally, thank you so much for developing this program!
